### DISCUSSION TITLE
custom domain dont work

### DISCUSSION THREAD

**Discussion by aliiiforghani** [vercel/community#3387] [Help]
### Summary

hi . 
i add custom domain to my next js and node js project but both of them dont work. proroo.ir api.proroo.ir

### Example

proroo.ir

### Steps to Reproduce

hi . 
i add custom domain to my next js and node js project but both of them dont work. proroo.ir api.proroo.ir

**Comment by amyegan** [comment]
Hey @aliiiforghani. I think there was just a normal delay in DNS record propagation. It looks like your domain is working correctly now 🙂
